woocommerce/woocommerce-android
# HACK Week: Add pagination support to attribute terms view

⚠️ Depends of https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2848 and https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2850

Why
==========
Fix for issue https://github.com/woocommerce/woocommerce-android/issues/9583. 

The current attributes term fetching doesn't use any pagination strategy and the `per_page` default value is 10. Because of this limitation, some users report problems seeing all terms of an attribute. 

How
==========
This PR solves this problem by allowing pagination when interacting with any Global attributes configuration under the Variable Product feature.

To avoid making this PR way too big, unit tests are being introduced in https://github.com/woocommerce/woocommerce-android/pull/9754

Captures
==========
https://github.com/woocommerce/woocommerce-android/assets/5920403/29ed4d90-f8f2-4e3d-8755-f58890bfce81

How to Test
==========
1. Log in to a store that has more than 10 options for an attribute type in a variable product (e.g. 25 different colors)
2. Create New Variable Product > Add Variations > Select existing attribute (e.g. Color)
3. Scroll down on the attribute screen to select an existing option
4. See that *ALL* options are loaded and displayed for that attribute as you scroll through them

Update release notes:
